### ISSUE TITLE
[Bug] Can not parse to read Chinese characters

### ISSUE THREAD

**Issue by Natmat626** [IncantaGames/unreal-bp-csv-parsing#1]
try to read Chinese characters get Garbled code

**Comment by mikeseese** [comment]
Non-ASCII characters are not supported in the 3rd party library we're using (https://github.com/vincentlaucsb/csv-parser). I have no plans to fix this, but I will leave this issue open if someone wants to make a fix and provide a tested and well-written PR.
